src/css/docs-prose.css

```diff
@@ -127,10 +91,6 @@
   }
 }
 
-.txt-spacing0 {
```

Comment:
@katydecorah I couldn't find any instances of this class being used in the library so thought we could remove it.